pipelines.adoc

```diff
@@ -161,7 +177,7 @@ oc apply -f <pipeline.yaml>
 
 === Run the pipeline
 
-A sample `manual-pipeline-run-template.yaml` file is provided in the link:https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator/manual-pipeline-runs[`/pipelines/manual-pipeline-runs` directory]. Rename the template file to `pipeline-run.yaml`, for example, and update the file to replace `application-stack-name` with the name of your application stack. After you update the file, run it as shown in the following example.
+A sample `manual-pipeline-run-template.yaml` file is provided in the link:kabanero-pipelines/pipelines/sample-helper-files/[`/pipelines/sample-helper-files` directory]. Rename the template file to `pipeline-run.yaml`, for example, and update the file to replace `application-stack-name` with the name of your application stack. After you update the file, run it as shown in the following example.
```

Comment:
Change
Rename the template file to `pipeline-run.yaml`, for example, and
to
Rename the template file to a name of your choice (for example, `pipeline-run.yaml`), and